staging/src/k8s.io/legacy-cloud-providers/aws/aws.go

```diff
@@ -1610,6 +1644,25 @@ func extractNodeAddresses(instance *ec2.Instance) ([]v1.NodeAddress, error) {
 		}
 	}
 
+	// IPv6. Ordered after IPv4 addresses, so legacy code can continue to just use the "first" address.
+	for _, networkInterface := range instance.NetworkInterfaces {
+		// skip network interfaces that are not currently in use
+		if aws.StringValue(networkInterface.Status) != ec2.NetworkInterfaceStatusInUse {
+			continue
+		}
+
+		for _, addr6 := range networkInterface.Ipv6Addresses {
+			if ipAddress := aws.StringValue(addr6.Ipv6Address); ipAddress != "" {
+				ip := net.ParseIP(ipAddress)
```

Comment:
the linter will complain here, you have to use the utilnet.ParseIPSloppy parses

Reply:
thanks @aojea 

fixed!